Documentation/botr/ryujit-overview.md

```diff
@@ -70,27 +70,29 @@ For this snippet of code (extracted from [tests/src/JIT/CodeGenBringUpTests/DblR
 
 A stripped-down dump of the `GenTree` nodes just after they are imported looks like this:
 
-    ▌ stmtExpr  void  (top level) (IL 0x000...0x026)
-    │        ┌──▌ lclVar    double V00 arg0
-    │     ┌──▌ *         double
-    │     │  └──▌ dconst    double 2.00
-    │  ┌──▌ /         double
-    │  │  │  ┌──▌ mathFN    double sqrt
-    │  │  │  │  │     ┌──▌ lclVar    double V02 arg2
-    │  │  │  │  │  ┌──▌ *         double
-    │  │  │  │  │  │  │  ┌──▌ lclVar    double V00 arg0
-    │  │  │  │  │  │  └──▌ *         double
-    │  │  │  │  │  │     └──▌ dconst    double 4.00
-    │  │  │  │  └──▌ -         double
-    │  │  │  │     │       lclVar    double V01 arg1
-    │  │  │  │     └──▌ *         double
-    │  │  │  │        └──▌ lclVar    double V01 arg1
-    │  │  └──▌ +         double
-    │  │     └──▌ unary -   double
-    │  │        └──▌ lclVar    double V01 arg1
-    └──▌  =         double
-       └──▌ indir     double
-          └──▌ lclVar    byref  V03 arg3
+```
+▌  STMT      void  (IL 0x000...0x026)
+└──▌  ASG       double
```

Comment:
Could you please add an additional comment about ASG? Which oper is src and which is dst.

Reply:
Can you help me understand what's so special about `ASG`? Sure, it's a rather weird operator but at the end of the day it's just a binary operator where operands are displayed like for all other binary operators - op1, op2. And the meaning of the operands is also pretty natural - LHS, RHS - exactly like they are written in high level languages.

Reply:
I agree that one can reason about it and come to the right conclusion, but `ASG` is weird, in that most people don't think of assignment as a binary operator.

Reply:
I've expanded the "Reading expression trees section" to include some more targeted examples, including assignment.